.gitattributes

```diff
@@ -6,51 +6,49 @@
 # The above will handle all files NOT found below
 #
 # These files are text and should be normalized (Convert crlf => lf)
-*.bat           text=auto
-*.css           text=auto
-*.df            text=auto
-*.htm           text=auto
-*.html          text=auto
-*.java          text=auto
-*.js            text=auto
-*.json          text=auto
-*.jsp           text=auto
-*.jspf          text=auto
-*.md            text=auto
-*.properties    text=auto
-*.tld           text=auto
-*.txt           text=auto
-*.xml           text=auto
-*.lst           text=auto
-*.ftl           text=auto
-*.gradle        text=auto
+*.css text=auto
```

Comment:
Oh I see. I made this changes in formatting based on Git documentation related to the format of those commands and using **one** space.